lib/Doctrine/ODM/PHPCR/Query/Builder/ConstraintFactory.php

```diff
@@ -127,7 +138,7 @@ public function fullTextSearch($field, $fullTextSearchExpression)
      * Same document constraint.
      *
      * <code>
-     * $qb->where()->same('/path/to/doc', 'sel_1');
+     * $qb->where()->same('/path/to/doc', 'sel_1')->end();
```

Comment:
i know, but we have them in some points but not others. i thought having them everywhere makes it easier to copy-paste fragments. do we have a doc section explaining when end() is needed? and does absolutely everything have end() or do we need to be more explicit with that?